tests/v2/validation/deleting/delete_cluster_rke1_test.go

```diff
@@ -0,0 +1,65 @@
+package deleting
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/provisioning"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	password "github.com/rancher/rancher/tests/framework/extensions/users/passwordgenerator"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type RKE1ClusterDeleteTestSuite struct {
+	suite.Suite
+	client  *rancher.Client
+	session *session.Session
+}
+
+func (c *RKE1ClusterDeleteTestSuite) TearDownSuite() {
+	c.session.Cleanup()
+}
+
+func (c *RKE1ClusterDeleteTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	c.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(c.T(), err)
+
+	c.client = client
+
+	enabled := true
+	var testuser = namegen.AppendRandomString("testuser-")
+	var testpassword = password.GenerateUserPassword("testpass-")
+	user := &management.User{
+		Username: testuser,
+		Password: testpassword,
+		Name:     testuser,
+		Enabled:  &enabled,
+	}
+
+	newUser, err := users.CreateUserWithRole(client, user, "user")
+	require.NoError(c.T(), err)
+
+	newUser.Password = user.Password
+}
+
+func (c *RKE1ClusterDeleteTestSuite) TestDeletingRKE1Cluster() {
+	clusterID, err := clusters.GetClusterIDByName(c.client, c.client.RancherConfig.ClusterName)
+	require.NoError(c.T(), err)
+
+	clusters.DeleteRKE1Cluster(c.client, clusterID)
+	provisioning.VerifyDeleteRKE1Cluster(c.T(), c.client, clusterID)
```

Comment:
curious if you tried verifyDeleteRKE2K3SCluster here at all, or do you have to use v3api here?

Reply:
They have to be separate as the `watchInterface` are different for v3 and v1. There probably is a way to consolidate it all, but that would mean a larger change across other files IMO.